.dev/revdep_get_repos.R

```diff
@@ -50,12 +50,20 @@ git_urls <- extract_github_repo(urls)
 matched <- nzchar(git_urls)
 
 utils::write.csv(
-  data.frame(package = lintr_pkg[!matched], repo = file.path("https://github.com/cran", lintr_pkg[!matched])),
+  data.frame(
```

Comment:
we don't need to do this in .dev scripts. I hope all our maintainers are on 4.0 by now :)

Reply:
Yes, possibly. But is there any harm in doing so? 🙃

Without it, if a future PR changes anything about this file, the build will fail because of this lint.

Reply:
we should probably just exclude .dev from .lintr_new

Reply:
(another good use case for context-specific config... would be great to turn off that specific linter in this directory)